openedx/features/caliper_tracking/transformers/user_settings_transformers.py

```diff
@@ -0,0 +1,41 @@
+"""
+Transformers for all the user setting events
+"""
+
+
+def edx_user_settings_changed(current_event, caliper_event):
+    """
+    Event occur when user change something in its Account Settings Page.
```

Comment:
`Event occurs when a user changes in his/her account settings page.`

Reply:
done

Reply:
`Event occurs when a user changes anything in his/her account settings page.`

Reply:
done